docs/contrib/authoring-slider-texts.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Writing slider descriptions
+---
+
+Sliders are the visual representation of [Inputs](inputs.md) in the ETM. Modellers can add a description to sliders. This description becomes visible when users select the information button presented as a question mark next to each slider. For clarity, it is important that all slider descriptions follow the same structure.
+
+## Getting started
+A good slider description contains the following aspects:
+1. Informs users about the slider functionality
+2. Shortly explains what type of technique is modelled with the slider (use full name to explain abbreviation if necessary, especialy if we use the abbreviation in a slidername)
+3. Indicates which other sliders affect the properties of this technology (including links to sliders)
+4. Indicates where users might find more detailed information (links to documentation or ref-manager)
+
+Slider descriptions should explicitly not contain the follwoing: 
+* Long explanation on how to set the slider, this should be covered in the documentation
+* Technical and financial properties
+* Region specific information
+* Links to non-ETM websites
+* Scenario-specific information
+
+## Form
+
+* **Language**: English / Dutch with a preference for British English.
+* **User Type**: Adult energy 'experts' or those aspiring to attain expertise in the subject matter. Focus on users already familiar with the topic. Avoid long explanations of the technique or trends, rather focus on what is meant by this technique (short definition) and how this technique is modelled (if relevant).
+* **Tone**: Formal yet constructive and activating. Use active and concise sentences. Prioritize readability, avoiding unnecessary technical jargon, and aim to nudge users toward understanding and utilising the ETM effectively.
+* **Style**: Visual emphasis, utilise elements like arrows for brevity. Consistency in style to enable users to quickly scan and locate specific information.
+
+To be avoided:
+* Saying “we” (ETM is an “it”)
+* Questions (“What do you think will happen…?”)
+* Avoid adressing the user with “you” 
+* Avoid abbreviations ("etc.")
```

Comment:
I would rename this documenation page to `authoring-texts`, "Writing texts", and then move this part to the beginning of the page. Some of the other information is also generic (applies to slides, slider texts, documentation and charts), whereas some of the other information is specific for slider texts. I would restructure this page to make that distinction more clearly.